src/calculateRentalCost.js

```diff
@@ -27,8 +27,25 @@
  * @return {number}
  */
 
+const PRICE_PER_DAY = 40;
+const DISCOUNT_FOR_SHORT_TERM = 20;
+const DISCOUNT_FOR_LONG_TERM = 50;
+
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_ALL_TERM = PRICE_PER_DAY * days;
```

Comment:
It is customary to call immutable static constants (numbers, periods) uppercase, and if the variable is computational, it is better to call it camelCase:
```suggestion
  const finalPrice = PRICE_PER_DAY * days;
```